oci/oci.go

```diff
@@ -500,18 +513,20 @@ func (r *Runtime) UpdateStatus(c *Container) error {
 			if err != nil {
 				return fmt.Errorf("status code conversion failed: %v", err)
 			}
-			c.state.ExitCode = int32(statusCode)
+			c.State.ExitCode = int32(statusCode)
 		}
 	}
 
+	c.toDisk()
```

Comment:
> If I understand toDisk() correctly, you're unconditionally storing the whole container structure.
> If we're only concerned with storing the state changes, we could easily do so from UpdateStatus() only when the container state actually changes.

of course I meant: I shouldn't be storing everything but `c.State` which I'm going to fix in the code